Zoo/src/JosPlateau/AnimalsRunner.java

```diff
@@ -0,0 +1,14 @@
+package JosPlateau;
+
+/**
+ * Created by admin-batch-a on 7/16/17.
+ */
+public class AnimalsRunner {
+
+    public static void main(String[] args) {
+
+        Animals yourList= new Animals ();
+        yourList.getMyAnimalList();
+        System.out.println(yourList);
```

Comment:
You cannot print isnatnce variable on the console. I need to see your thought process in these codes. I could explain more if need be.